<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboardcontrol</class>
 <widget class="QFrame" name="Dashboardcontrol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">QScrollArea{
background-repeat: no-repeat;
background-position: center;
border:none;

}

 QScrollBar:horizontal
 {

     height: 15px;

     margin: 3px 15px 3px 15px;

     border: 1px transparent #2A2929;

     border-radius: 4px;


 }



 QScrollBar::handle:horizontal

 {

     background-color: white;      /* #605F5F; */

     min-width: 5px;

     border-radius: 4px;

 }



 QScrollBar::add-line:horizontal

 {

     margin: 0px 3px 0px 3px;

     border-image: url(:/qss_icons/rc/right_arrow_disabled.png);

     width: 10px;

     height: 10px;

     subcontrol-position: right;

     subcontrol-origin: margin;

 }



 QScrollBar::sub-line:horizontal

 {

     margin: 0px 3px 0px 3px;

     border-image: url(:/qss_icons/rc/left_arrow_disabled.png);

     height: 10px;

     width: 10px;

     subcontrol-position: left;

     subcontrol-origin: margin;

 }



 QScrollBar::add-line:horizontal:hover,QScrollBar::add-line:horizontal:on

 {

     border-image: url(:/qss_icons/rc/right_arrow.png);

     height: 10px;

     width: 10px;

     subcontrol-position: right;

     subcontrol-origin: margin;

 }





 QScrollBar::sub-line:horizontal:hover, QScrollBar::sub-line:horizontal:on

 {

     border-image: url(:/qss_icons/rc/left_arrow.png);

     height: 10px;

     width: 10px;

     subcontrol-position: left;

     subcontrol-origin: margin;

 }



 QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal

 {

     background: none;

 }





 QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal

 {

     background: none;

 }



 QScrollBar:vertical

 {

     

	background-color: rgb(49, 51, 50);

     width: 15px;

     margin: 15px 3px 15px 3px;

     border: 1px transparent #2A2929;

     border-radius: 4px;

 }



 QScrollBar::handle:vertical

 {

     background-color: white;         /* #605F5F; */

     min-height: 5px;

     border-radius: 4px;

 }



 QScrollBar::sub-line:vertical

 {

     margin: 3px 0px 3px 0px;

     border-image: url(:/qss_icons/rc/up_arrow_disabled.png);

     height: 10px;

     width: 10px;

     subcontrol-position: top;

     subcontrol-origin: margin;

 }



 QScrollBar::add-line:vertical

 {

     margin: 3px 0px 3px 0px;

     border-image: url(:/qss_icons/rc/down_arrow_disabled.png);

     height: 10px;

     width: 10px;

     subcontrol-position: bottom;

     subcontrol-origin: margin;

 }



 QScrollBar::sub-line:vertical:hover,QScrollBar::sub-line:vertical:on

 {



     border-image: url(:/qss_icons/rc/up_arrow.png);

     height: 10px;

     width: 10px;

     subcontrol-position: top;

     subcontrol-origin: margin;

 }





 QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on

 {

     border-image: url(:/qss_icons/rc/down_arrow.png);

     height: 10px;

     width: 10px;

     subcontrol-position: bottom;

     subcontrol-origin: margin;

 }



 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical

 {

     background: none;

 }





 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical

 {

     background: none;

 }</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>797</width>
        <height>498</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <widget class="QWidget" name="page_AppList"/>
         <widget class="QWidget" name="page_VersionList"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
